server/events/command_runner.go

```diff
@@ -238,9 +239,12 @@ func (c *DefaultCommandRunner) getAzureDevopsData(baseRepo models.Repo, pullNum
 	return pull, headRepo, nil
 }
 
-func (c *DefaultCommandRunner) buildLogger(repoFullName string, pullNum int) *logging.SimpleLogger {
-	src := fmt.Sprintf("%s#%d", repoFullName, pullNum)
-	return c.Logger.NewLogger(src, true, c.Logger.GetLevel())
+func (c *DefaultCommandRunner) buildLogger(repoFullName string, pullNum int) logging.SimpleLogging {
+
+	return c.Logger.WithHistory(
+		"repository", repoFullName,
```

Comment:
what about "repo"?

Reply:
Naming is hard.

Reply:
just thinking it's shorter, don't really mind.